client/src/partials/users/user.modal.js

```diff
@@ -0,0 +1,76 @@
+angular.module('bhima.controllers')
+  .controller('UserModalController', UserModalController);
+
+UserModalController.$inject = ['$state', '$uibModal', 'ProjectService', 'UserService', 'NotifyService'];
+
+function UserModalController($state, $uibModal, Projects, Users, Notify) {
+  var vm = this;
+
+  // the user object that is either edited or created
+  vm.user = {};
+  vm.isCreating = $state.params.creating;
+
+  //exposed methods
+  vm.submit = submit;
+  vm.closeModal = closeModal;
+  vm.validPassword = validPassword;
+  vm.setPasswordModal = setPasswordModal;
+
+  Projects.read().then(function (data) {
+    vm.projects = data;
+  })
+  .catch(Notify.handleError);
+
+  if(!vm.isCreating){
+
+    Users.read($state.params.id)
+      .then(function (user) {
+        vm.user = user;
+      })
+      .catch(Notify.handleError);
+  }
+
+  // submit the data to the server from all two forms (update, create)
+  function submit(userForm) {
+
+    if (userForm.invalid) { return; }
```

Comment:
The `invalid` property does not exist on AngularJS forms.  It's called `$invalid`.  [Documentation](https://docs.angularjs.org/api/ng/type/form.FormController#$invalid).